core/events/events_marker_move.ts

```diff
@@ -96,4 +97,11 @@ export class MarkerMove extends UiBase {
   }
 }
 
+export interface MarkerMoveJson extends AbstractEventJson {
+  isCursor: boolean;
+  blockId: string|null;
```

Comment:
in the click events, `blockId` is optional rather than nullable, should we be consistent here?